sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -511,7 +552,29 @@ object ScalaJSPlugin extends AutoPlugin {
     Seq(
         scalaJSStage := Stage.FastOpt,
         scalaJSUseRhinoInternal := false,
-        ScalaJSPluginInternal.scalaJSClearCacheStats := globalIRCache.clearStats()
+        ScalaJSPluginInternal.scalaJSClearCacheStatsInternal := {},
+
+        // Clear the IR cache stats every time a sequence of tasks ends
+        onComplete := {
+          val prev = onComplete.value
+
+          { () =>
```

Comment:
Why not use the `andThen` thingy here?

Reply:
Because

> `andThen` is not a member of `() => Unit`

(`andThen` is only defined on `Function1`, but `onComplete` is a `Function0`)